src/utils/notify/helper/constructData.js

```diff
@@ -0,0 +1,39 @@
+function constructData(entityTypeId, entityId, actorId) {
+  if (entityTypeId >= 1 && entityTypeId <= 5)
+    return {
+      click_action: 'OPEN_PROJECT_ACTIVITY',
+      data: {
+        project_id: entityId,
+      },
+    };
+  if (entityTypeId >= 6 && entityTypeId <= 10)
+    return {
+      click_action: 'OPEN_IDEA_ACTIVITY',
+      data: {
+        idea_id: entityId,
+      },
+    };
+  if (entityTypeId >= 11 && entityTypeId <= 15)
+    return {
+      click_action: 'OPEN_THOUGHT_ACTIVITY',
+      data: {
+        thought_id: entityId,
+      },
+    };
+  if (entityTypeId === 17)
```

Comment:
Did you mean entityTypeId 26 here? because 17 is for accepting connection request